packages/discovery-react-components/src/components/DiscoverySearch/DiscoverySearch.tsx

```diff
@@ -197,6 +209,7 @@ const fieldsStoreDefaults: FieldsStore = {
 export const searchContextDefaults = {
   aggregationResults: null,
   searchResponseStore: searchResponseStoreDefaults,
+  globalAggregationsResponseStore: globalAggregationsResponseStoreDefaults,
```

Comment:
does this replace `aggregationResults` or is it an entirely different set of data? i can't quite remember unfortunately

Reply:
Ah, yes, thanks. I meant to remove this. Updated without `aggregationResults` and also updated the `overrideAggregationResults` to work with the new `globalAggregationsResponseStore`.